spec/features/ncr/work_orders/approve_spec.rb

```diff
@@ -21,6 +21,21 @@
     end
   end
 
+  scenario "allows delegate to approve work order" do
+    Timecop.freeze do
```

Comment:
Here's the commit when those tests went in https://github.com/18F/C2/commit/73a4d386 which contains a comment with a link that hints at why the Timecop stuff is there.

I would leave them as is. It's lamentable there are not better comments as to why, but it obviously had some purpose at the time or @nickbristow wouldn't have gone to the trouble.